src/main/java/eu/enexa/service/EnexaServiceImpl.java

```diff
@@ -54,13 +63,15 @@ public Model startExperiment() {
         }
         // do not use experiment.getLocalName() it will remove the first character !
 
-        sharedDirLocalPath = sharedDirLocalPath + File.separator + experiment.getURI().replace("http://", "");
+        sharedDirLocalPath = sharedDirLocalPath +File.separator+appName+ File.separator + experiment.getURI().split("/")[experiment.getURI().split("/").length -1];
```

Comment:
I think this is the third time that I see this line, in which you create a path for a specific experiment. Maybe a single function (e.g., in the java-utils library) would be good. In that case, you 
1. have the implementation only once,
2. can easily write JUnit tests for it, to ensure that it works as expected, and
3. others can use it in the future :wink: